src/rubberbandmodel.cpp

```diff
@@ -233,3 +239,14 @@ void RubberbandModel::setVectorLayer( QgsVectorLayer* layer )
 
   emit vectorLayerChanged();
 }
+
+
+bool RubberbandModel::frozen() const
+{
+  return mFrozen;
+}
+
+void RubberbandModel::setFrozen(const bool &frozen )
+{
+  mFrozen = frozen;
```

Comment:
emit?

Reply:
if there is no use of the signal, shall I rather remove it (the signal)?

Reply:
It's in the property. And you can bind qml properties to this.

Reply:
And better add a

```
if ( mFrozen == frozen )
  return;
```

It avoids deadlocks... Just in case.

Reply:
fixed